docs/architecture/_index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Viam Architecture"
+linkTitle: "Architecture"
+weight: 409
+type: "docs"
+description: "How a machine running on Viam is structured, from on-device to cloud communications."
+imageAlt: "Viam architecture"
+images: ["/viam/machine-components.png"]
+tags: ["components", "services", "communication"]
+menuindent: true
+---
+
+## `viam-server`
+
+`viam-server` is the open-source executable binary that runs on your machine's SBC or other computer.
+
+`viam-server` does the following locally:
+
+- Runs drivers for your hardware
+- Runs motion planning and vision services
+- Runs {{< glossary_tooltip term_id="module" text="modules" >}}
+- Manages local connections between all these resources
+- Captures and stores data
+
+When `viam-server` can connect to the cloud, it also:
+
+- Automatically pulls configuration updates you make in the Viam app
+- Gets new versions of software packages
+- Uploads and syncs image and sensor data
+- Allows you to remotely control your machine from the Viam app
+
+`viam-server` can use the internet (WAN) or local networks (LAN) to establish peer-to-peer connections between two {{< glossary_tooltip term_id="machine" text="machines" >}}, or to a client application.
+
+## Components, services, modules
+
+A [_component_](/components/) represents a physical piece of hardware in your {{< glossary_tooltip term_id="machine" text="machine" >}}, and the software that directly supports that hardware.
+
+A [_service_](/services/) is a software package that makes it easier to add complex capabilities such as motion planning or object detection to your machine.
+
+Viam has many built-in components and services that run within `viam-server`.
+
+A [_modular resource_](/registry/) is a custom component or service, not built into `viam-server` but rather provided by a _module_ that you or another user have created.
+A module runs in parallel to `viam-server` on your machine, communicating over UNIX sockets, and `viam-server` manages its lifecycle.
```

Comment:
I don't really think you say "runs in parallel". I don't know if it's started as a child process or something. I think we need to probably check that
```suggestion
A module runs as a process managed by `viam-server` on your machine, communicating over UNIX sockets, and `viam-server` manages its lifecycle.
```

Reply:
I have checked this, it is run as a child process. So very much not in parallel.
```suggestion
A module runs as a child process managed by`viam-server` on your machine, communicating over UNIX sockets, and `viam-server` manages its lifecycle.
```